### DISCUSSION TITLE
Kernel.create_function_from_prompt() missing 2 required positional arguments: 'function_name' and 'plugin_name'

### DISCUSSION THREAD

**Discussion by samo9789** [microsoft/semantic-kernel#5307] [Q&A] [answered]
Hi,

I have just upgraded my code to use semantic_kernel==0.9.0b1 and follow the documentation in the sample notebookx
Most say ` self.kernel.create_function_from_prompt(prompt_template_config=self.chat_prompt_template_config)
`
But when I have this line, I get this error
`TypeError: Kernel.create_function_from_prompt() missing 2 required positional arguments: 'function_name' and 'plugin_name'`
Can anyone tell me what I am doing wrong ? 

Thank you

**Comment by moonbox3** [comment, marked as answer]
Hi @samo9789, we've fixed this in subsequent versions. We're currently on 0.9.4b1. Please upgrade when possible. Also, we'll have more eyes on this if it is filed next time in the GitHub issues versus the Discussions section. Thanks.

**Comment by pchao04** [comment]
Same error, just use the get start sample: https://github.com/microsoft/semantic-kernel/blob/main/python/README.md

**Comment by jsuarez-dev** [comment]
Same error, on the "get started" sample. if you check on the current version 0.9.4b1 on the function [`create_function_from_prompt`](https://github.com/microsoft/semantic-kernel/blob/main/python/semantic_kernel/kernel.py) the args `function_name` and `plugin_name` are used. Maybe an easy fix should be updating the "get started" sample.

Something like :
```python
    function = kernel.create_function_from_prompt(
        function_name="tldr_function",
        plugin_name="tldr_plugin",
        prompt_template_config=prompt_template_config,
    )
```
 and the second part to something like:

```python
summarize = kernel.create_function_from_prompt(
        function_name="tldr_function",
        plugin_name="tldr_plugin",
        prompt="{{$input}}\n\nOne line TLDR with the fewest words.",
        prompt_template_settings=req_settings,
)
```
